src/Loggers/LogImage.jl

```diff
@@ -1,11 +1,11 @@
 
-@enum ImageFormat HW WH HWC WHC CHW CWH HWN WHN NHW NWH HWCN WHCN CHWN CWHN NHWC NWHC NCHW NCWH
+@enum ImageFormat L CL LC HW WH HWC WHC CHW CWH HWN WHN NHW NWH HWCN WHCN CHWN CWHN NHWC NWHC NCHW NCWH
 """
     log_images(logger::TBLogger, name::AbstractString, imgArrays::AbstractArray, format::ImageFormat; step = nothing)
 
 Log multiple images using `Array` of images and format
 - imgArrays: `Array` of images, e.g. Array{Array{Float64, 3}, 1}. `Array` of images can be multidimensional.
-- format: format which applies to each image in the `Array` of images. It can be one of {HW, WH, HWC, WHC, CHW, CWH, HWN, WHN, NHW, NWH, HWCN, WHCN, CHWN, CWHN, NHWC, NWHC, NCHW, NCWH}
+- format: format which applies to each image in the `Array` of images. It can be one of {L, CL, LC, HW, WH, HWC, WHC, CHW, CWH, HWN, WHN, NHW, NWH, HWCN, WHCN, CHWN, CWHN, NHWC, NWHC, NCHW, NCWH}
```

Comment:
Can we have a link to some docs explaining what they mean?

Reply:
okay i'll explain it in the docstring. that should do it? 
>C = Color? H= Height, W=Width, L=Length?, N=observation?  
this is correct

Reply:
Yep